bin/pycbc_inspiral

```diff
@@ -201,9 +201,22 @@ parser.add_argument("--multiprocessing-nprocesses", type=int,
                          "Used in conjunction with the option"
                          "--finalize-events-template-rate which should be set"
                          "to a multiple of the number of processes.")
-
 # Add options groups
 psd.insert_psd_option_group(parser)
+parser.add_argument("--use-precomputed-psd", action="store_true", default=False,
+                    help='Use compressed waveforms from the bank file.')
+
+# FIXME: We may need to use multiple files based on chunk if those are not
+# generated in the workflow.
+# psd.add_argument('--psd-files', metavar='FILE', nargs='*',
+#                         help='Instead of time-independent PSDs, use time-varying '
+#                         'PSDs from the given HDF5 files and pick the appropriate '
+#                         'PSD for each injection')
+
+parser.add_argument("--precomputed-psd-file", metavar='FILE', type=str,
+                        help='Instead of time-independent PSDs, use time-varying '
+                        'PSDs from the given HDF5 files and pick the appropriate '
+                        'PSD for each injection')
```

Comment:
The argument only takes one option, but the help mentions multiple PSD files.

I think it would be best not to mention injections - I cant think of alternative use-cases, but there might be one!

Reply:
In fact it may be worth adding this to the `insert_psd_option_group` as this may be useful elsewhere, though I dont know the code well enough to say thats definitely possible

Reply:
Done